tests/test_room_resource/test_update_assigned_resource.py

```diff
@@ -0,0 +1,57 @@
+import os
+import sys
+
+from tests.base import BaseTestCase, CommonTestCases
+from fixtures.room_resource.update_assigned_resource import (
+    update_assigned_resource_query,
+    expected_update_assigned_resource_query,
+    update_with_negative_quantity,
+    update_non_existing_room,
+    update_non_existing_resource
+)
+from fixtures.room.assign_resource_fixture import (
+    assign_resource_mutation
+)
+sys.path.append(os.getcwd())
+
+
+class TestUpdateRoomResorce(BaseTestCase):
+
+    def test_update_assigned_resource_by_non_admin(self):
```

Comment:
Please provide documentation for this and other tests

Reply:
This has been worked on